Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Internal/CustomerCancellationDues.hs

```diff
@@ -153,8 +156,6 @@ customerCancellationDuesSync merchantId merchantCity apiKey req = do
                 }
         QCC.create cancellationCharges
```

Comment:
I was talking about the atomicity of the API. If it throws error, no db insertion/updation should happen

Reply:
In which case is it happening, all the insert , updates are after the errors ?